postman/dashboards/postman-dashboard.json

```diff
@@ -0,0 +1,210 @@
+{
+    "name": "Postman Monitor Overview",
```

Comment:
You will need to run the dashboard sanitizer on this dashboard as there are account IDs and permissions in the JSON. 

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboard-json